README.md

```diff
@@ -1,50 +1,184 @@
+[relaxed-caps](https://github.com/appium/appium/tree/master/packages/relaxed-caps-plugin) | With the advent of Appium 2.0, the Appium server begins to require that all capabilities conform to the W3C requirements for capabilities. Among these requirements is one that restricts capabilities to those found in a predetermined set. Appium supports many additional capabilities as extension capabilities, and these must be accessed with the prefix appium: in front of the capability name. There are a lot of test scripts out there that don't conform to the requirement, and so this plugin is designed to make it easy to keep running these scripts even with the new stricter capabilities requirements beginning with Appium 2.0. Basically, it inserts the appium: prefix for you!
+[universal-xml](https://github.com/appium/appium/tree/master/packages/universal-xml-plugin) | This is an official Appium plugin designed to make XML source retrieved from iOS and Android use the same node and attribute names, to facilitate cross-platform test writing.
+
+#### Plugins Provided By Third Parties
+
+These plugins are supported by different companies or communities.
+Please report any issues related to their functionality to their
+respective maintainers. If you are a developer yourself and have created a new useful
+plugin then don't hesitate to create a PR and add a link to your plugin into the below list.
+
+<details>
+  <summary>Plugins List</summary>
+
+  Name | Description
+  --- | ---
+  [appium-device-farm](https://github.com/AppiumTestDistribution/appium-device-farm) | This is an Appium plugin designed to manage and create driver session on connected android devices and iOS Simulators.
+</details>
+
+### Server Command Line Interface
+
+In order to start sending commands to Appium over the wire it must be listening
+on the URL where your client library expects it to listen.
+Use the following commands to run and configure Appium server:
+
+```bash
+# Start the server on the default port and host (e.g. http://0.0.0.0:4723/)
+appium server
+# Start the server on the given port, host and use the base path prefix (the default prefix is /)
+appium server -p 9000 -a 127.0.0.1 -pa /wd/hub
+
+# Get the list of all supported command line parameters.
+# This list would also include descriptions of driver-specific
+# command line arguments for all installed drivers.
+# Each driver and plugin must have their command line arguments
+# exposed in a special JSON schema declared as a part of the corresponding
+# package.json file.
+appium server --help
+```
+
+Appium supports execution of parallel server processes as well as parallel driver sessions within
+single server process. Refer the corresponding driver documentations regarding which mode is optimal
+for the particular driver or whether it supports parallel sessions.
 
 ### Why Appium?
 
 1. You don't have to recompile your app or modify it in any way, due
```

Comment:
we should probably update this to `You usually don't have to`, since Espresso is kind of borderline and other drivers might work differently too.